test/axiom/new_meta_chain.js

```diff
@@ -87,6 +88,7 @@ contract('Axiom::newMetaChain', (accounts) => {
 
   contract('Negative Tests', () => {
     it('should fail when called by non technical governance address', async () => {
+      newMetaChainParams.txOptions.from = accountProvider.get();
```

Comment:
We have `Object.freeze(newMetaChainParams);` at line 86.
I'm not sure if this works.

We have 2 options:
1. Remove `Object.freeze(newMetaChainParams);` from line 86, keep this code and remove the `Object.assign` in the following line. We have to do this at multiple places in the test.
2. If it does not work then remove this line.

Reply:
Done